ReactAndroid/src/main/java/com/facebook/react/devsupport/WindowOverlayCompat.java

```diff
@@ -0,0 +1,18 @@
+package com.facebook.react.devsupport;
+
+import android.os.Build;
+import android.view.WindowManager;
+
+/**
+ * Compatibility wrapper for apps targeting API level 26 or later.
+ * See https://developer.android.com/about/versions/o/android-8.0-changes.html#cwt
+ */
+class WindowOverlayCompat {
+
+  private static final int ANDROID_OREO = 26;
+  private static final int TYPE_APPLICATION_OVERLAY = 2038;
```

Comment:
Cannot use the Android constants because react native doesn't compile to v26 yet (RN currently uses v23)